weaver/.github/workflows/deploy_docs.yml

```diff
@@ -15,11 +15,11 @@ on:
   workflow_dispatch:
 
 jobs:
-  deploy-docs:
+  publish-docs:
```

Comment:
Let's disable this action for now, since the publication happens to the Labs doc pages. We need to refactor the docs configuration to publish to a common Hyperledger Cacti site.

Reply:
This is in `weaver/.github` directory so its disabled for now. Only workflows in `.github` are active.